interface/patient_file/summary/add_edit_issue.php

```diff
@@ -702,55 +686,6 @@ function divclick(cb, divid) {
   </td>
  </tr>
 
- <!-- For Athletic Teams -->
-
- <tr<?php if (! $GLOBALS['athletic_team']) echo " style='display:none;'"; ?> id='row_injury_grade'>
-  <td valign='top' nowrap><b><?php echo xlt('Grade of Injury'); ?>:</b></td>
-  <td>
-<?php
-echo generate_select_list('form_injury_grade', 'injury_grade', $irow['injury_grade'], '');
-?>
-  </td>
- </tr>
-
- <tr<?php if (! $GLOBALS['athletic_team']) echo " style='display:none;'"; ?> id='row_injury_part'>
-  <td valign='top' nowrap><b><?php echo xlt('Injured Body Part'); ?>:</b></td>
```

Comment:
i think the !globals is for the style . the Injured body part is the athletic code